### ISSUE TITLE
Composer и документация

### ISSUE THREAD

**Issue by mazurva** [lepism/BemPHP#2]
Composer: https://getcomposer.org/
Было бы проще подключать библиотеку к проектам, и существенно расширило количество пользователей и контрибьюторов. Жду появления на https://packagist.org/
Однако свой автозагрузчик не убирайте, а то есть разные проекты и разработчики.
Добавьте статью с хабра в качестве README_ru, а после вполне возможно, кто нить и переведен на другие языки.

**Comment by lepism** [comment]
Да, спасибо, мне в комментариях про это не раз сказали. Уже изучаю матчасть. В 1.2 реализую.
В README хотел добавить, но забыл. Спасибо, что напомнили.

**Comment by mazurva** [comment]
Поставьте себе Composer и запустите в терминале/консоли

```
composer init
```

Мастер вам поможет создать первый composer.json, тем более что пока у Вас никаких зависимостей нет в проекте.

**Comment by corpsee** [comment]
https://github.com/lepism/BemPHP/pull/3
